src/views/components/address-year-from-field.njk

```diff
@@ -1,14 +1,14 @@
-{% from "hmpo-text/macro.njk" import hmpoText %}
+{% macro yearFromField(ctx, params) %}
+  {% from "hmpo-text/macro.njk" import hmpoText %}
 
-<h4>{{translate("fields.addressYearFrom.title")}}</h4>
```

Comment:
The title should still be there from the Figmas? This would remove it for UK journey as well.